db/model/sample.js

```diff
@@ -370,11 +347,14 @@ module.exports = function sample(seq, dataTypes) {
 
     }, // hooks
     indexes: [
-      { unique: true, fields: ['aspectId', 'subjectId', 'isDeleted'] },
       {
-        name: 'SampleStatusDeletedAt',
+        name: 'AspectIdSubjectId',
+        unique: true,
+        fields: ['aspectId', 'subjectId'],
+      },
+      {
+        name: 'SampleStatus',
         fields: [
-          'deletedAt',
           'status',
```

Comment:
AspectIdSubjectId and SampleStatus are old indices. AspectIdSubjectId was named aspectid_subjectid_isDeleted and SampleStatus was named SampleStatusIdDeleted. They were using the isDeleted and deletedAt fields, and I have removed these fields from the indices.